app/assets/stylesheets/views/_browse.scss

```diff
@@ -26,6 +26,17 @@ $browse-header-background-colour: #263135;
   overflow: auto;
 }
 
+.browse__breadcrumb-border {
+  background: govuk-colour("blue");;
```

Comment:
Nitpick: need to remove the extra semi-colon.

Reply:
Thanks, will do!